srcpkgs/youtube-dl/template

```diff
@@ -10,10 +10,10 @@ checkdepends="flake8 python3-nose"
 short_desc="CLI program to download videos from YouTube and other sites"
 maintainer="Daniel Eyßer <daniel.eysser@gmail.com>"
 license="Unlicense"
-homepage="https://ytdl-org.github.io/youtube-dl/"
-changelog="https://raw.githubusercontent.com/ytdl-org/youtube-dl/master/ChangeLog"
-distfiles="https://yt-dl.org/downloads/${version}/youtube-dl-${version}.tar.gz"
-checksum=ac1a799cf968345bf29089ed2e5c5d4f4a32031625d808369e61b6362d1c7cde
+homepage="https://yt-dl.org"
+changelog="https://gitlab.com/dstftw/youtube-dl/-/raw/master/ChangeLog"
+distfiles="https://youtube-dl.org/downloads/${version}/${pkgname}-${version}.tar.gz"
```

Comment:
`homepage` and `distfiles` doesn't use the same website!

Reply:
fixed!